autopilot.go

```diff
@@ -4,67 +4,44 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/cloudfoundry/cli/plugin"
+	"github.com/xchapter7x/autopilot/application_repo"
 	"github.com/xchapter7x/autopilot/rewind"
 )
 
-func fatalIf(err error) {
-	if err != nil {
-		fmt.Fprintln(os.Stdout, "error:", err)
-		os.Exit(1)
-	}
+var ActionList []rewind.Action
```

Comment:
as stated in other comments, we probably don't want to export this as we can more accurately test using provided fakes.